pkg/image/dryrun_client.go

```diff
@@ -18,12 +18,57 @@ package image
 
 import (
 	"github.com/sirupsen/logrus"
+	"strings"
 )
 
 // DryRunClient is an implementation of Client that logs the image operations that would
 // be performed rather than performing them.
 type DryRunClient struct{}
 
+const v1_19_images = `docker.io/library/httpd:2.4.38-alpine
```

Comment:
I did not expect to see this hard coded list of images... seeing this PR is to solve that.  Is this const for a test ?

Reply:
ah yeah it's just for testing sonobuoy, this is a dryrun client which doesn't actually execute Docker and doesn't test the cluster.